README.md

```diff
@@ -25,6 +25,10 @@ Rsync cookbook with rsyncd LWRP. More info on ryncd options can be found in the
 
 The name of the init service
 
+`node['rsyncd']['init']` _(String) default: "sysvinit(for Debian platform and Redhat platform without systemd)/systemd (for Redhat platform supporting systemd )"_
```

Comment:
Ideally we don't use an attribute for this since this isn't something a user should be modifying. We're moving away from using attributes as global variables since it makes it very hard to understand what the true tuneables actually are.